src/components/alert/alert.ios.scss

```diff
@@ -3,90 +3,208 @@
 $alert-ios-message-text-color:                  inherit !default;
+
+/// @prop - Padding of the alert empty message
 $alert-ios-message-padding-empty:               0 0 12px 0 !default;
 
+/// @prop - Max height of the alert content
 $alert-ios-content-max-height:                  240px !default;
 
+/// @prop - Margin top of the alert input
 $alert-ios-input-margin-top:                    10px !default;
+
+/// @prop - Padding on the alert input
 $alert-ios-input-padding:                       6px !default;
+
+/// @prop - Border color of the alert input
 $alert-ios-input-border-color:                  #ccc !default;
+
+/// @prop - Border of the alert input
 $alert-ios-input-border:                        $hairlines-width solid $alert-ios-input-border-color !default;
+
+/// @prop - Border radius of the alert input
 $alert-ios-input-border-radius:                 4px !default;
+
+/// @prop - Background color of the alert input
 $alert-ios-input-background-color:              #fff !default;
 
+/// @prop - Flex wrap of the alert button group
 $alert-ios-button-group-flex-wrap:              wrap !default;
 
+/// @prop - Flex of the alert button
 $alert-ios-button-flex:                         1 1 auto !default;
+
+/// @prop - Margin of the alert button
 $alert-ios-button-margin:                       0 !default;
+
+/// @prop - Min width of the alert button
 $alert-ios-button-min-width:                    50% !default;
+
+/// @prop - Min height of the alert button
 $alert-ios-button-min-height:                   44px !default;
+
+/// @prop - Font size of the alert button
 $alert-ios-button-font-size:                    17px !default;
+
+/// @prop - Color of the text in the alert button
 $alert-ios-button-text-color:                   color($colors-ios, primary) !default;
+
+/// @prop - Background color of the alert button
 $alert-ios-button-background-color:             transparent !default;
+
+/// @prop - Background color of the alert activated button
 $alert-ios-button-background-color-activated:   #e9e9e9 !default;
 
+/// @prop - Border width of the alert button
 $alert-ios-button-border-width:                 $hairlines-width !default;
+
+/// @prop - Border style of the alert button
 $alert-ios-button-border-style:                 solid !default;
+
+/// @prop - Border color of the alert button
 $alert-ios-button-border-color:                 #dbdbdf !default;
+
+/// @prop - Border radius of the alert button
 $alert-ios-button-border-radius:                0 !default;
 
+/// @prop - Font weight of the main text on the alert button
 $alert-ios-button-main-font-weight:             bold !default;
 
+/// @prop - Border top of the alert list
 $alert-ios-list-border-top:                     $alert-ios-button-border-width $alert-ios-button-border-style $alert-ios-button-border-color !default;
 
+/// @prop - Padding on the label for the radio alert button
 $alert-ios-radio-label-padding:                 13px !default;
```

Comment:
Starting here to the end of the file the word "button" is added to the radio and checkbox descriptions. Could you remove this or change it to "input"? This is also in the other mode files. :)